docs/api/rabbitmq.com.ref.asciidoc

```diff
@@ -298,7 +298,7 @@ Spec is the desired state of the RabbitmqCluster Custom Resource.
 | Field | Description
 | *`replicas`* __integer__ | Replicas is the number of nodes in the RabbitMQ cluster. Each node is deployed as a Replica in a StatefulSet. Only 1, 3, 5 replicas clusters are tested. This value should be an odd number to ensure the resultant cluster can establish exactly one quorum of nodes in the event of a fragmenting network partition.
 | *`image`* __string__ | Image is the name of the RabbitMQ docker image to use for RabbitMQ nodes in the RabbitmqCluster. Must be provided together with ImagePullSecrets in order to use an image in a private registry.
-| *`imagePullSecrets`* __link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#localobjectreference-v1-core[$$LocalObjectReference$$] array__ | List of Secret resource containing access credentials to the registry for the RabbitMQ image. Required if the docker registry is private.
```

Comment:
This particular line shouldn't be modified, though the autogen keeps trying to for some reason... We should probably separately look into why this happens

Reply:
Fixed, thank you.

Reply:
@MarcialRosales we shouldn't change this line. See Connor's comment.

Reply:
Fixed in https://github.com/rabbitmq/cluster-operator/pull/846/commits/a231127c53cca2479cdebcf822ee85fe9837d5f9